crates/nu-command/tests/commands/url/path.rs

```diff
@@ -0,0 +1,28 @@
+use nu_test_support::{nu, pipeline};
```

Comment:
Do you think we should keep `url path` and `url query`? My immediate reaction is that they should not exist if we have `url parse` since one should just do e.g. `($url | url parse).path`.